### PR TITLE
doc: update dashboard readme

### PR DESCRIPTION
**Pull request by minteliuwm**
## Brief change log

- update dashboard readme doc

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by shidayang** [comment]
@YesOrNo828 cc. Flink ci occasionally fails to pass

**Review by shidayang — APPROVED** [review]

**Comment by shidayang** [comment]
Thanks for your contributions. This pr LGTM

**Review by baiyangtx — APPROVED** [review]
LGTM,

Thanks a lot for your contribution.

File: ams/dashboard/README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+ - See the License for the specific language governing permissions and 
+ - limitations under the License.
+-->
+
+## Requirements
+
+- node `16.18.0`
+- yarn `1.22.19`
+
+## Installation
```
Comment (zhoujinsong) [inline ams/dashboard/README.md:24]: Perhaps we can organize the content according to this directory：
```
## Installing dependencies
## Running Locally
## Building project
```

This way may be clearer and make it easier for everyone to find the content they are interested in.

File: ams/dashboard/README.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and 
+ - limitations under the License.
+-->
+
+## Requirements
+
+- node `16.18.0`
```
[the remark this answers is not in the document]
Comment (minteliuwm) [reply ams/dashboard/README.md]: Due to an anomaly in the compilation of higher versions of Node, Node version `16.x` is recommended until this compatibility issue is resolved.

File: ams/dashboard/README.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+- yarn `1.22.19`
```
[the remark this answers is not in the document]
Comment (minteliuwm) [reply ams/dashboard/README.md]: Not required, added optional note

File: ams/dashboard/README.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+
+### Configure the ams Server to connect to
+
+1. open the `vue.config.js` under the folder `ams/dashboard`
+
+2. find the `ENV_HOST` configuration and change the environment corresponding to the `DEV`
+
+```
+const ENV_HOST = {
+  DEV: 'http://192.168.3.10:1630/', // modify it
```
[the remark this answers is not in the document]
Comment (minteliuwm) [reply ams/dashboard/README.md]: Of course, I've modified it here.

File: ams/dashboard/README.md (hunk 1 above)
Comment (minteliuwm) [reply ams/dashboard/README.md]: Okay, I rearranged it according to the new table of contents.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**